wiki/Shortcut_key_reference/nl.md

```diff
@@ -0,0 +1,290 @@
+---
+tags:
+  - hotkey
+  - hotkeys
+  - keyboard
+  - shortcuts
+---
+
+# Sneltoets combinaties
+
+Osu! heeft een groot aantal sneltoets combinaties die kunnnen worden gebruikt in-game of wanneer je op een bepaalt scherm bent. De combinaties en hun functies zijn hieronder vermeld.
+
+*Opmerking: Dit artikel geeft de standaard sneltoets combinaties weer. Je kunt enkele hiervan veranderen in de [opties](/wiki/Options).*
```

Comment:
*Opmerking: Dit artikel geeft de standaard sneltoetscombinaties weer. Je kunt enkele hiervan wijzigen bij de [opties](/wiki/Options).*

Reply:
'in de opties' lijkt mij beter.